xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -1638,7 +1638,7 @@ bool CGUIDialogVideoInfo::GetSetForMovie(const CFileItem *movieItem, CFileItemPt
     return false;
 
   CFileItemList listItems;
-  std::string baseDir = "videodb://movies/sets/";
+  std::string baseDir = "videodb://movies/sets/?includesetswithonemovie=true";
```

Comment:
Shouldn't the baseDir, as the name states, be the base without including any further options? And the option be injected by the functionality that actually requires that option?

Reply:
> actually the only place where the options should be used..

yes, actually it's the only place. we have  calls to `videodb://movies/sets/` wide spread throughout the codebase. when that happens (without added option) the functionality doesn't change. we need this option when filling the movieset-editor, because it should always return "single-movie-sets", so they can be fed with more movies. would definitely add a comment at this point. renaming? hm, matter of taste i think, but no problem if forced ;)

Reply:
I agree that the variable name `baseDir` is not ideal. Unforunately it's used throughout a lot of old code.